<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SWidget</class>
 <widget class="QWidget" name="SWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wWrapper" native="true">
     <property name="styleSheet">
      <string notr="true">#wWrapper {
	background-color:rgba(230, 230, 230, 100);
	border: 0.5px solid rgba(198, 198, 198, 180);
	border-radius: 3px;

}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QWidget" name="wContainer" native="true">
        <property name="styleSheet">
         <string notr="true">#wContainer {
	background-color:rgba(188, 188, 188, 150);
	border-radius: 3px;
}

/*rgba(237, 241, 230, 100);*/</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="wToolBar" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#wToolBar {
	background-color: rgba(103, 103, 103, 180);
	border-radius: 3px;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="wMenu" native="true">
              <property name="styleSheet">
               <string notr="true">#wMenu {
	background-color: transparent;
	border-radius: 3px;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="wWindowControl" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {	
	border: 0px solid black;
	border-radius: 3;
	background-color: transparent;
	text-align: center;
}

#btClose{ image: url(:icons/icons/close.png);}
#btClose::hover{ image: url(:icons/icons/close_on.png);}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>5</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btMinimize">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>15</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>--</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btClose">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>15</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>15</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
           <zorder>wWindowControl</zorder>
           <zorder>horizontalSpacer</zorder>
           <zorder>wMenu</zorder>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wBody" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="wTop" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#wTop {
	background-color: rgba(103, 103, 103, 180);
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {

	background-color: transparent;
	border: 0px;
}

#btArtistView{ image: url(:/icons/icons/artist_2.png)}
#btArtistView::hover{ image: url(:/icons/icons/artist_on.png)}

#btSongView{ image: url(:/icons/icons/song.png);}
#btSongView::hover{ image: url(:/icons/icons/song_on.png);}

#btGenreView{ image:url(:/icons/icons/genre.png);}
#btGenreView::hover{ image:url(:/icons/icons/genre_on.png);}</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="btArtistView">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>15</width>
                      <height>15</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btGenreView">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>15</width>
                      <height>15</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btSongView">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>15</width>
                      <height>15</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="wCenter" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="topMargin">
                <number>10</number>
               </property>
               <item>
                <widget class="QListWidget" name="listWidget">
                 <property name="styleSheet">
                  <string notr="true">#listWidget{
	border: 0px solid grey;
    background-color: transparent;	 
    show-decoration-selected: 0;
	padding-left: 20px;
   	padding-right: 20px;
}

#listWidget::item:selected {
	 border: none;
 	 background-color:rgba(240, 240, 240, 100);
     color: black;     
 }</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="showDropIndicator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>50</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="flow">
                  <enum>QListView::LeftToRight</enum>
                 </property>
                 <property name="isWrapping" stdset="0">
                  <bool>true</bool>
                 </property>
                 <property name="resizeMode">
                  <enum>QListView::Adjust</enum>
                 </property>
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="viewMode">
                  <enum>QListView::IconMode</enum>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="tableWidget">
                 <property name="styleSheet">
                  <string notr="true">#tableWidget{
background-color: transparent;
margin-left: 20px;
margin-right: 20px;

}
QHeaderView{
	background-color: rgb(220, 220, 220);/*rgb(132, 132, 132);*/
border: 0px;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
}
QHeaderView::section {
     background-color: transparent;
	 color: gray;
 }
</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="showDropIndicator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="dragDropOverwriteMode">
                  <bool>false</bool>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <property name="showGrid">
                  <bool>false</bool>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                 <property name="cornerButtonEnabled">
                  <bool>true</bool>
                 </property>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string/>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Song</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Artist</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Album</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Length</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Genres</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Quality</string>
                  </property>
                  <property name="textAlignment">
                   <set>AlignHCenter|AlignVCenter|AlignCenter</set>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
